### ISSUE TITLE
Help pvp enjoyers please

### ISSUE THREAD

**Issue by maxcodk** [MarcelineVQ/TankPlates#10]
Hello, is it possible to add a way for auto/optional disabling addon in pvp? It is hard to understand what classes you face and also when enemies stop tartgeting you their nameplates get red, using pfui.

**Comment by MarcelineVQ** [comment]
Yeah it's quite reasonable to have a toggle for it, I'll look into that soon as I want to revisit the addon to fix spell targeting weirdness anyway
